perf/benchmark/runner/runner.py

```diff
@@ -41,15 +41,18 @@
 def pod_info(filterstr="", namespace=NAMESPACE, multi_ok=True):
     cmd = "kubectl -n {namespace} get pod {filterstr} -o json".format(
         namespace=namespace, filterstr=filterstr)
-    op = getoutput(cmd)
-    o = json.loads(op)
+    completed_process = subprocess.run(shlex.split(cmd), check=True, encoding="utf-8")
```

Comment:
<img width="1439" alt="image" src="https://github.com/istio/tools/assets/19339970/3300a634-5f04-4212-af60-a4cf40bdb5f3">

It seems that this commit will result bug.